packages/arktype-validator/src/index.ts

```diff
@@ -29,6 +29,7 @@ export const arktypeValidator = <
   schema: T,
   hook?: Hook<T['infer'], E, P>
 ): MiddlewareHandler<E, P, V> =>
+  // @ts-expect-error not typed well
```

Comment:
Looking at the other `@hono/*-validator` packages, this is a common issue that is fixed with this directive